docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```
@@ -2141,7 +2534,7 @@ content: |
   % elif test_run_output.ready():
   ${ test_run_output.get() }
   % else:
-  Sorry, ALKilnInThePlayground is unable to get the console logs.
+  Sorry, ALKilnInThePlayground is unable to get the\nconsole logs yet. Refreshing the page later might or might not help.
```

Is this still true now that I've fixed the revocation problem? I think not, so suggesting this change:

```suggestion
  Sorry, ALKilnInThePlayground is unable to get the console logs.
```